### ISSUE TITLE
(Api Live Dev) 500 response Internal Server Error...but then it eventually works

### ISSUE THREAD

**Issue by dkershner6** [serverless-stack/serverless-stack#1163]
I am not sure how to word this into an issue, but I did my best to search and didnt find anything similar, thought I would post here and see if anyone has experienced similar:

SST 0.54.2
Ubuntu Linux Server 20.04
~32GB Ram free (just to ensure this isnt it)

I have tried this on a couple stacks, both exhibit the problem
sst start works well, deploys, and it takes maybe 2 minutes for live lambda dev to start.
Send request, 500 after exactly 10 seconds. I have confirmed through logs that this is the lambda timing out.
Nothing occurs for maybe 10 minutes, then the logs show in the console for the original request, and at the end it says
```
3b89a65f-77fa-483e-a134-3e98cdd45cb3 RESPONSE null
7c0136f7-d14e-4168-86e9-e6543e9e616e-1639698942879 Failed to send response because the Lambda function is disconnected
```

After this point, everything works well, as I am used to. Responses and logs are immediate.

**Comment by dkershner6** [comment]
Troubleshooting session went well, @thdxr should have this fix out soon

**Comment by thdxr** [comment]
Fixed in 0.54.3! Closing this issue
